README.md

```diff
@@ -7,12 +7,13 @@ Cypress runner for GoCD that:
 
 # Prerequisites
 
-Cypress npm package is installed in project.
+This package supports Cypress starting from version 10.
```

Comment:
Too bad that we no longer provide at least best-effort support for older Cypress versions, because, for example, previously the library just worked well with Cypress 9 as well and some projects may need more time to upgrade to v10, which is a big leap as it comes with bigger compatibility breakages than ever. Although there are multiple ways to deal with this problem, like just maintaining 2.x releases a bit longer to support such projects, probably it's not the best idea to support too old versions due to potential vulnerabilities coming from unmaintained package and dependency versions. But Cypress 9 could be a good candidate to support for a while like that.

Reply:
"Edit2: We could mention in the README, like in the form of a table, which releases support which Cypress version. What do you think?" 

Yes, that is a good point thanks, I will update that.